packages/rocketchat-lib/server/lib/sendEmailOnMessage.js

```diff
@@ -167,17 +167,18 @@ RocketChat.callbacks.add('afterSaveMessage', function(message, room) {
 
 		if (usersOfMention && usersOfMention.length > 0) {
 			usersOfMention.forEach((user) => {
+				const allowNotificationsChange = RocketChat.settings.get('Accounts_AllowEmailNotifications');
```

Comment:
I'd rename this to `emailNotificationsAllowed` or something that reads more intuitively.  I knew what it did but got had to keep referring back up